legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -193,6 +193,15 @@ class FilingTitles(str, Enum):
         'title': 'Registrars Order',
         'displayName': "Registrar's Order",
         'code': 'NOFEE'},
+    'registration': {
+        'name': 'registration',
+        'title': 'Registration',
+        'displayName': 'Registration',
```

Comment:
IIs this supposed to be a  generic Registration. I think this has to be specific for SP and GP

Reply:
There is a separate ticket for that

Reply:
ok